pkg/config/node.go

```diff
@@ -91,6 +92,7 @@ type ClusterLBConfig struct {
 	ApiLBIPs     []net.IP
 	ApiIntLBIPs  []net.IP
 	IngressLBIPs []net.IP
+	PlatformType string
```

Comment:
I would rather see `PlatformType string` defined inside `type Cluster struct` because platform is a property of the cluster, not of the loadbalancer